mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterActor.java

```diff
@@ -759,23 +762,35 @@ public GetActiveJobsRequest(ClusterID clusterID) {
         }
     }
 
+    interface HasAttributes {
```

Comment:
nit: this name reads like a bool.

Reply:
Hmm, this is just like a trait, and traits are usually named this way.